packages/core/content-manager/server/src/controllers/single-types.ts

```diff
@@ -73,8 +72,8 @@ const createOrUpdateDocument = async (ctx: any, opts?: { populate: object }) =>
   }
 
   const sanitizedBody = await sanitizeFn(body);
-  return entityManager.update(documentVersion || { id: otherDocumentVersion.documentId }, model, {
-    data: sanitizedBody,
+  return singleTypes.createOrUpdate(model, {
```

Comment:
we probably can rename it update as it's know clearly 2 different APIs like in the doc service for consistency